kuma/static/js/font-check.js

```diff
@@ -0,0 +1,43 @@
+var mdn = window.mdn || {};
+
+(function(mdn) {
+    'use strict';
+
+    var fonts = [
+        {
+            'name' : 'Open Sans Light',
+            'className' : 'ffo-opensanslight',
+            'loaded': false,
+            'varient' : [
+                {'weight' : 'normal'},
+                {'weight' : 'bold'},
+                {'style' : 'italic'}
+            ]
+        },
+        {
+            'name' : 'Open Sans',
+            'className' : 'ffo-opensans',
+            'loaded': false,
+            'varient' : [
+                {'weight' : 'normal'},
+                {'weight' : 'bold'},
+                {'style' : 'italic'}
+            ]
+        }
+    ];
+
+    for (var i = 0; i < fonts.length; i++) {
```

Comment:
Not a lot of iterations here probably but, would it be worth to cache the length of the fonts array?

```
for (var i = 0, l = fonts.length; i < l; i++) {
```